.github/workflows/_build.yaml

```diff
@@ -134,6 +134,12 @@ jobs:
             BASE_IMAGE=${{ inputs.BASE_IMAGE }}
             BUILD_DATE=${{ inputs.BUILD_DATE }}
 
+      - name: Save container tags (mealkit and final) for publishing step
+        run: |
+          CONTAINER_TAG_NAME=$(echo ${{ inputs.CONTAINER_NAME }} | tr '[:lower:]' '[:upper:]' | tr '-' '_')
+          echo "TAG_${CONTAINER_TAG_NAME}_MEALKIT=${{ steps.mealkit-metadata.outputs.tags }}" | tee -a $GITHUB_OUTPUT
```

Comment:
Given that this is a generic and reusable workflow, its output should also have generic names.
The outputs from different calls to this workflow can be distinguished by their job name.
i.e. we want
```
${{ needs.build_A.outputs.TAG_MEALKIT }}
${{ needs.build_B.outputs.TAG_MEALKIT }}
```
instead of
```
${{ needs.build_A.outputs.TAG_A_MEALKIT }}
${{ needs.build_B.outputs.TAG_B_MEALKIT }}
```

Reply:
This is leftover from my experiments. This is step is not required any more. All tags will be saved inside `_ci.yaml` workflow under `save-tags` job